### PR TITLE
fix(sso) Retrieve cookie configs separately from SSO configs

### PR DESCRIPTION
**Pull request by chriscollins3456**
Grabs the configs for setting out Actor cookie through the SSO from the configs object instead of trying to shoe-horn them into the SSO configs object. This fixes an issue where if you try to configure SSO through the UI and have these env variables set, we still read these values properly.

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by aditya-radhakrishnan — COMMENTED** [review]
I'm wondering if we should create a new object that contains these cookie configs. What do you think?

**Comment by chriscollins3456** [comment]
> I'm wondering if we should create a new object that contains these cookie configs. What do you think?

100% agree! sorry should've done that in the first place

File: datahub-frontend/app/auth/CookieConfigs.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  public static final String AUTH_COOKIE_SAME_SITE = "play.http.session.sameSite";
+  public static final String DEFAULT_AUTH_COOKIE_SAME_SITE = "LAX";
+  public static final String AUTH_COOKIE_SECURE = "play.http.session.secure";
+  public static final boolean DEFAULT_AUTH_COOKIE_SECURE = false;
+
+  private final int _ttlInHours;
+  private final String _authCookieSameSite;
+  private final boolean _authCookieSecure;
+
+  public CookieConfigs(final Config configs) {
```
Comment (aditya-radhakrishnan) [inline datahub-frontend/app/auth/CookieConfigs.java:18]: Awesome!

**Review by aditya-radhakrishnan — APPROVED** [review]
LGTM! Thanks for making this refactor

**Pull request merged**